### DISCUSSION TITLE
idea: super famidash

### DISCUSSION THREAD

**Discussion by Rubiidev-18** [tfdsoft/famidash#147] [Ideas]
Basically a famidash snes port with mode support so we can truly rotate the icon, have more colors and more detailed backgrounds, etc

**Comment by zephyrside** [comment]
snes doesn't natively support sprite rotation, the super fx is used for that

not to mention a PC Engine port would be more practical since cc65 supports compiling to it (cc65 doesn't support the 65816)

**Comment by kandowontu** [comment]
> snes doesn't natively support sprite rotation, the super fx is used for that
> 
> not to mention a PC Engine port would be more practical since cc65 supports compiling to it (cc65 doesn't support the 65816)

no way :O 

We are maybe thinking about doing a super famidash wayyyy down the line when we're done with this. All depends on the teams interest at that point!
